File: Code/CH6 Code/VB/GridRestrict-vb/Town.vb
Option Strict On

Public Class Town

  Private mName As String
  Private mCounty As String
  Private mState As String
  Private mMayor As String
  Private mZip As String
  Private mMillRate As Single

  Public Property Name() As String
    Get
      Return mName
    End Get
    Set(ByVal Value As String)
      mName = Value
    End Set
  End Property

  Public Property County() As String
    Get
      Return mCounty
    End Get
    Set(ByVal Value As String)
      mCounty = Value
    End Set
  End Property

  Public Property State() As String
    Get
      Return mState
    End Get
    Set(ByVal Value As String)
      mState = Value
    End Set
  End Property

  Public Property Mayor() As String
    Get
      Return mMayor
    End Get
    Set(ByVal Value As String)
      mMayor = Value
    End Set
  End Property

  Public Property Zip() As String
    Get
      Return mZip
    End Get
    Set(ByVal Value As String)
      mZip = Value
    End Set
  End Property

  Public Property MillRate() As Single
    Get
      Return mMillRate
    End Get
    Set(ByVal Value As Single)
      mMillRate = Value
    End Set
  End Property


End Class

File: Code/CH6 Code/C#/GridRestrict-c/Town.cs
using System;

namespace GridRestrict_c
{
	/// <summary>
	/// This class contains information about a town.
	/// </summary>
  public class Town
  {
    private string mName;
    private string mCounty;
    private string mState;
    private string mMayor;
    private string mZip;
    private float  mMillRate;

    public string Name
    {
      get { return mName; }
      set { mName = value; }
    }

    public string County
    {
      get { return mCounty; }
      set { mCounty = value; }
    }

    public string State
    {
      get { return mState; }
      set { mState = value; }
    }

    public string Mayor
    {
      get { return mMayor; }
      set { mMayor = value; }
    }

    public string Zip
    {
      get { return mZip; }
      set { mZip = value; }
    }

    public float MillRate
    {
      get { return mMillRate; }
      set { mMillRate = value; }
    }

  }
}
